db/migrate/20230510154419_add_index_to_posts_user_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToPostsUserId < ActiveRecord::Migration[7.0]
+  def change
+    add_index :posts, :author_id
+  end
+end
```

Comment:
![image](https://github.com/AyoMoses1/Gems_Blog/assets/93039359/7030a14d-0edb-4ec1-ad5e-1a9312ba37b3)

**[OPTIONAL]**

- [ ] It would be nice if you remove the last 5 migrations because your indexes are already created by the rails app so you don't need these migrations so kindly remove them to make your code clean 👍